README.md

```diff
@@ -129,6 +157,18 @@ The `is_final` field is always `True` because no updates will be output for the
 
     Example "captions" responses can be found in [examples/responses/captions.md](https://github.com/verbit-ai/verbit-streaming-python-sdk/blob/main/examples/responses/captions.md).
 
+### Error handling and recovery
+
+#### Initial connection
+In case the WebSocket client fails to establish the (initial) connection with the service (e.g. due to temporary unavailability), 
```

Comment:
remove parentheses - both instances are important information, not need to demote them using parentheses